modules/hal_gigadevice/CMakeLists.txt

```diff
@@ -5,51 +5,61 @@ if(CONFIG_HAS_GD32_HAL)
 
 string(TOUPPER ${CONFIG_SOC} gd32_soc_uc)
 
-set(gd32_soc_dir     ${ZEPHYR_HAL_GIGADEVICE_MODULE_DIR}/${CONFIG_SOC_SERIES})
-set(gd32_cmsis_dir   ${gd32_soc_dir}/cmsis/gd/${CONFIG_SOC_SERIES})
+set(gd32_soc_dir     ${ZEPHYR_HAL_GIGADEVICE_MODULE_DIR}/${CONFIG_GD32_SOC_SERIES})
```

Comment:
update cd799d8abc9dc1afe99bebe4e69c8bfff277f4fe commit message (mentions outdated `gd32_soc_series_lc`)

Reply:
Sorry, I forgotten it!